typefaces.md

```diff
@@ -1,5 +1,7 @@
 # Guide on Choosing a Font
-![Font Guide](https://github.com/gyuden/mms142-groupd-2021/blob/main/Font-Styles-Guide.jpg?raw=true)
+![Font Guide](https://github.com/gyuden/mms142-groupd-2021/blob/main/Font-Styles-Guide.jpg?raw=true)\
+![Font Pairing](https://github.com/gyuden/mms142-groupd-2021/blob/main/FontPairing-CheatSheet.jpg?raw=true)\
+![Font Anatomy and Layout](https://github.com/gyuden/mms142-groupd-2021/blob/main/TheAnatomyofTypography-Poster-01.jpg?raw=true)
 
```

Comment:
Much better we mix combinations of serif, sans serif and a little bit of script fonts